decidim-core/db/seeds.rb

```diff
@@ -102,8 +102,30 @@
       es: "<p>Descripción</p>",
       ca: "<p>Descripció</p>"
     },
+    active: true,
     start_date: 1.month.ago.at_midnight,
     end_date: 2.months.from_now.at_midnight,
     participatory_process: participatory_process1
   )
+
+  Decidim::ParticipatoryProcessStep.create!(
+    title: {
```

Comment:
`seeds` are executed in the third party's environment - i.e. another deployment with a customized `I18n.available_locales` variable. This means we can be potentially breaking the seeds in that context. We have to find a way around that by creating a helper that populates all the `available_locales` from `Faker` or something similar.